lib/common/SidetreeError.ts

```diff
@@ -4,7 +4,7 @@
  */
 export default class SidetreeError extends Error {
   constructor (public code: string, message?: string) {
-    super(message ? `${code}: ${message}` : code);
+    super(message ? message : code);
```

Comment:
This means that the when someone calls `sidetreeError.message`, it will not have the code there right. Not sure why would we make this change?

Reply:
You are right `code` would not be in `message` with this change, but I ask myself, if they want to know the `code`, the more realistic scenario would be that they would just get it from `sidetreeError.code`, so no additional parsing required. So no real benefit for having the code in both properties. Also, `message` is intended to be a more detailed, user friendly message, so when such message is given, I feel we should just use it as intended and not do additional morphing.